testing/sliding-sync-integration-test/src/lib.rs

```diff
@@ -282,7 +282,7 @@ async fn adding_list_later() -> anyhow::Result<()> {
     let build_list = |name| {
         SlidingSyncList::builder(name)
             .sync_mode(SlidingSyncMode::Selective)
-            .set_range(0u32, 10u32)
+            .set_range(0u32..=10u32)
```

Comment:
```suggestion
            .set_range(0..=10)
```

well, those tests have been removed, so a rebase will remove the need to update them.